File: Kosmeetuper/Models/Feedback.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Kosmeetuper.Models
{
    public class Feedback
    {
        public int Id { get; set; }

        [Required(ErrorMessage = "Name is required.")]
        public string Name { get; set; }
        
        [Required(ErrorMessage = "Email is required.")]
        public string Email { get; set; }

        [Required(ErrorMessage = "Message is required.")]
        public string MessageDescription { get; set; }

        public DateTime CreatedAt{ get; set; }

    }
}

File: server/models/feedback.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const feedbackSchema = new Schema({
    name: {
        type: String,
        required: "Name is Required",
    },
    email: {
        type: String,
        required: "Email is Required",
        // match: [/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/],
      },
    message: {
        type: String,
        required: "Message is Required"
    }
})

module.exports = mongoose.model("Feedback", feedbackSchema);